src/annotator/components/NotebookModal.js

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useRef, useState } from 'preact/hooks';
+import classnames from 'classnames';
+
+import { createSidebarConfig } from '../config/sidebar';
+
+// FIXME: use the button from the frontend shared package once this is stable.
+import Button from '../../sidebar/components/Button';
+
+/**
+ *
+ * @typedef NotebookIframeProps
+ * @prop {Record<string, any>} config
+ * @prop {string} groupId
+ */
+
+/**
+ * Create the iframe that will load the notebook application.
+ * 
+ * @param {NotebookIframeProps} props
+
+ */
```

Comment:
```suggestion
 * @param {NotebookIframeProps} props
 */
```

There is an empty line in this comment.